### ISSUE TITLE
Blank screen before face step when uploadFallback: false and no camera

### ISSUE THREAD

**Issue by anttilbc** [onfido/onfido-sdk-ui#1030]
### What was the expected behaviour?

Suggestion to switch to mobile. 

### What happened instead?

Blank screen

### Version info:

- Onfido SDK 5.7.0 - 5.9.2 (In 5.6.0 it worked as expected)
- OS tested on ubuntu and windows
- Browser tested firefox and chrome
- Device desktop, laptop

### Integration configuration:
```
Onfido.init({
    containerId: 'onfido-mount',
    token: token,
    steps: [
      'welcome',
      'document',
      {
        type: 'face',
        options: {
          uploadFallback: true,
        }
      },
      'complete',
    ],
    onComplete: function (data) {
        alert("Yes! Completed!")
    }
});
```

### Steps to reproduce:

Make sure camera is not available on your desktop/laptop. After uploading documents the selfie step should come up as blank screen with some errors found on console

### Demo:
https://jsfiddle.net/537m9qb8/

**Comment by anttilbc** [comment]
Hi,
I did some digging and it seems this part is responsible for the error:
https://github.com/onfido/onfido-sdk-ui/blob/686daf75515560395dba711f60e8787d3f7f9136/src/components/Capture/Face.js#L131-L133
It attempts to get the mobile OS name even when on desktop which causes the error string to be incorrect. I believe adding simple `!isDesktop` to the if statement would fix this

**Comment by stefaniacardenas** [comment]
Hi @anttilbc, thanks for reporting this. This issue has been addressed in [this PR](https://github.com/onfido/onfido-sdk-ui/pull/1014). The fix will go live in the next release, so probably next week.

**Comment by anttilbc** [comment]
Ah I didn't notice that. I'll try it out as soon as it is live :+1:

**Comment by stefaniacardenas** [comment]
This bug has been fixed in version [5.10.0](https://github.com/onfido/onfido-sdk-ui/releases/tag/5.10.0). I will close this issue now. Thanks for reporting it.
